packages/components/src/internal/util/jDateFormatParser.ts

```diff
@@ -25,6 +25,10 @@ const javaToMomentFormatMapping: FormatMapping = {
     hh: 'hh',
     H: 'H',
     HH: 'HH',
+    k: 'H',
```

Comment:
Seems like this could cause an unexpected shift by an hour when the Java format is 'k' or 'KK'?

Reply:
I don't believe so. It is definitely a bit misleading as the grid display will use k / KK but the date picket will use h / HH. So the date picker will have the user pick the time like "00:00" for midnight but the grid will display it as "24:00". However, I believe the correct value is actually being stored in this case from my testing.

<img width="362" alt="Screenshot 2024-06-26 at 8 55 27 AM" src="https://github.com/LabKey/labkey-ui-components/assets/6411206/30b8824f-8981-44c2-8249-f896623e6aa8">

Reply:
Just makes me wonder if we want/need to support the k/KK options, but that's more work and for a later discussion.